src/core/mac/mac_frame.hpp

```diff
@@ -1023,8 +1024,35 @@ class Frame : public otRadioFrame
      *
      */
     void SetCslIe(uint16_t aCslPeriod, uint16_t aCslPhase);
+
+    /**
+     * Indicates whether or not the CSL IE is present.
+     *
+     * @retval TRUE   If the CSL IE is present.
+     * @retval FALSE  If the CSL IE is not present.
+     *
+     */
+    bool IsCslIePresent(void) const;
```

Comment:
The child class `TxFrame` already has a method with the same name. But it's a different implementation. To avoid conflict and confusion, I suggest using a different name. Waht about `HasCslIe`.